config/bpfman-deployment/daemonset.yaml

```diff
@@ -210,6 +215,9 @@ spec:
         - hostPath:
             path: /proc
           name: host-proc
+        - hostPath:
+            path: /run/netns
+          name: host-netns
```

Comment:
what about `/var/run/netns` ? if this volume not mounted I don't think netobserv usecas will work

Reply:
/var/run/netns is a symbolic link to /run/netns, so it will contain the same netns files.